docs/release-notes/8.8.asciidoc

```diff
@@ -0,0 +1,97 @@
+** <<rules-api-get,Get rule>>
+** <<rules-api-find,Find rules>>
+** <<rules-api-create,Create rule>>
+** <<rules-api-update,Update rule>>
+** <<bulk-actions-rules-api,Bulk rule actions>>
+
+//end::breaking-changes[]
+
+
+[discrete]
+[[deprecations-8.8.0]]
+==== Deprecations
+There are no deprecations in 8.8.0.
+
+
+[discrete]
+[[features-8.8.0]]
+==== New features
+
+* Adds a new response action that allows you to execute commands on a selected host (https://github.com/elastic/security-team/issues/5441[5441]).
+* Adds a threat intelligence overview section to alert details for alerts generated by indicator rules ({pull}155328[#155328]).
+* Adds the `kibana.alert.url` field to alert events. Its value is a link to the alert details in Kibana ({pull}155069[#155069]).
+* Adds an option to duplicate a shared exception list from its more actions menu ({pull}154991[#154991]).
+* Adds a button that allows you to delete a note from a Timeline ({pull}154834[#154834]).
+* Adds rule action filters that let you specify whether actions should only run under particular conditions ({pull}154680[#154680]).
+* Adds a "snooze rule" button to the rules table ({pull}153083[#153083]).
+* Adds a "snooze rule" button to the rule details and edit rule pages ({pull}155407[#155407], {pull}155612[#155612]).
+* Adds controls to the Alerts page that allow you to customize which filters appear at the top of the page ({pull}152450[#152450]).
+* Adds an expandable section to the alert details flyout with various additional features and information. NOTE: to access this functionality you currently need to add a line to the `kibana.json` file. {pull}152935[#152935], {pull}152767[#152767], {pull}152303[#152303], {pull}152150[#152150], {pull}152047[#152047], {pull}150240[#150240], {pull}153807[#153807], {pull}153187[#153187], {pull}153706[#153706], {pull}153074[#153074], {pull}154196[#154196], {pull}153903[#153903], {pull}154114[#154114]).
+* Adds a "visualizations" section to the Alert details flyout ({pull}154786[#154786]).
+* Adds Analyzer to the expanded alert details flyout ({pull}153709[#153709]).
+* Adds Session View to the expanded alert details flyout ({pull}154597[#154597]).
+
+
+LENS update: visualizations throughout security have been updated to Lens visualizations, and can be opened in Lens
+
+[discrete]
+[[enhancements-8.8.0]]
+==== Enhancements
+
+* Updates the Notable anomalies table, and renames it to Anomalies ({pull}155687[#155687]).
+* Updates the Data Quality dashboard to include a new tree map, and storage size metrics for each index ({pull}155581[#155581]).
+* Displays additional anomaly jobs on the Entity Analytics dashboard ({pull}155520[#155520]).
+* Adds cloud infrastructure-related fields to the alert details flyout's highlighted fields ({pull}155247[#155247]).
+* Allows you to specify how to handle alert suppression for alerts with missing fields ({pull}155055[#155055]).
+* Updates the rule settings Actions interface ({pull}154526[#154526]).
```

Comment:
@e40pud should this be release-noted as a new feature? An enhancement is typically smaller in scale.

```suggestion
* Gives users more control over how they receive alert notifications and lets them define conditions that must be met for an notification to occur ({pull}154526[#154526]).
```